LocalSettings.php

```diff
@@ -358,16 +358,14 @@
 $wgGroupPermissions['chatmod']['chat'] = false;
 $wgGroupPermissions['autopatrol']['autopatrol'] = true;
 $wgGroupPermissions['interwiki-admin']['interwiki'] = true;
-$wgGroupPermissions['template-editor']['template-editor'] = true;
-$wgGroupPermissions['sysop']['template-editor'] = true;
 $wgAddGroups['sysop'] = array( 'editor', 'reviewer', 'autoreview', 'autopatrol', 'patroller', 'translateadmin', 'confirmed', 'researcher', 'chatmod', 'blockedfromchat', 'template-editor' );
```

Comment:
Let’s also remove template-editor from this array (line 361) as well as on line 362 please.

Reply:
@MacFan4000 Are lines 361-362 the `$wgAddGroups` lines? Assuming that's the case, I was just thinking that. 👍

Reply:
@Justarandomamerican, have you also removed `template-editor` permissions from any users that had it assigned on-wiki? Not sure if required, but I think it's a good practice.

Reply:
@dmehus I'll do that later.

Reply:
That will need to be done before I merge this, otherwise it won’t be possible.

Reply:
**Done**